tests/acceptance/course-page/complete-first-stage-test.js

```
@@ -6,6 +6,7 @@ import { animationsSettled, setupAnimationTest } from 'ember-animated/test-suppo
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
 import { signIn } from 'codecrafters-frontend/tests/support/authentication-helpers';
+import { currentURL } from '@ember/test-helpers';
```

ESLint

[error] 'currentURL' is defined but never used.

(no-unused-vars)